iset.mm

```diff
@@ -22409,14 +22409,14 @@ deduction version (Contributed by Thierry Arnoux, 30-Jan-2017.) $)
   $}
 
   ${
-    $d x y z $.  $d y z ph $.  $d x z ps $.
+    $d x y z $.  $d x z ph $.  $d y z ps $.
     sbralie.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
     $( Implicit to explicit substitution that swaps variables in a quantified
        expression.  (Contributed by NM, 5-Sep-2004.) $)
-    sbralie $p |- ( [ x / y ] A. x e. y ph <-> A. y e. x ps ) $=
-      ( vz cv wral wsb cbvralsv sbbii nfv raleq sbie bitri sbco2 ralbii ) ACDGZ
-      HZDCIZACFIZFCGZHZBDUBHZTUAFRHZDCIUCSUEDCACFRJKUEUCDCUCDLUAFRUBMNOUCUAFDIZ
-      DUBHUDUAFDUBJUFBDUBUFACDIBACDFAFLPABCDBCLENOQOO $.
+    sbralie $p |- ( [ y / x ] A. y e. x ph <-> A. x e. y ps ) $=
+      ( vz cv wral wsb cbvralsv sbbii nfv raleq sbie bitri sbco2 equcoms ralbii
+      wb ) ADCGZHZCDIZADFIZFDGZHZBCUDHZUBUCFTHZCDIUEUAUGCDADFTJKUGUECDUECLUCFTU
+      DMNOUEUCFCIZCUDHUFUCFCUDJUHBCUDUHADCIBADCFAFLPABDCBDLABSCDEQNOROO $.
```

Comment:
Why does this proof become so much longer?

Reply:
Because I made the cosmetic choice of keeping `x = y` in the hypothesis instead of switching into `y = x`. Do you prefer to have `y = x`, but with the original proof length? (For me it's fine both ways).

Some potentially useful data include the fact that currently `x = y` appears 486 times in iset.mm, while `y = x` appears only 19 times (and most of those 19 times are there to show commutation properties).

Reply:
@GinoGiotto thank you for the explanation. We should not switch the antecedent in the hypothesis (`x = y ` should be the preferred way), so we can keep the longer proof.

Reply:
I don't really have an opinion about these renamings, as long as they are **only** renamings.
When we write `x = y -> ( ph <-> ps )`, this generally goes with DV(x,ps) and/or DV(y,ph), so if you swap x and y, do it consistently. (I haven't checked the present case.)

Reply:
> When we write `x = y -> ( ph <-> ps )`, this generally goes with DV(x,ps) and/or DV(y,ph).

I didn't know that x is coupled with ps and y is coupled with ph. I'll try to fix this in a next PR.

Reply:
I do not know what @benjub wants to say with his comment, and I don't see any need for a fix/change: DV(x,ps) means that x does not occur in ps, but of course y can (and usually should) occur in ps, analogously with ph. So we (usually) have ps(y) and ph(x). If the theorem is valid without DV conditions, they should not be added.

Reply:
Of course, DV conditions should not be added if they are not necessary. What I mean is that when they are needed, it is usual to write that kind of hypotheses as `x = y -> (ph(x) <-> ps(y))` (or sometimes  `x = y -> (ph(x,y) <-> ps(y))` or `x = y -> (ph(x) <-> ps(x,y))`), where I made explicit the variables that may occur instead of the variables that do not.